tools/build.js

```diff
@@ -12,6 +12,9 @@ mkdirp('dist')
     .then(() => commandRunner('npm run build-editor'))
     .then(() => mkdirp('dist/editor'))
     .then(() => copy('./src/editor/dist/**', './dist/editor'))
+    // Temporary Need a better way of handling this
+    .then(() => copy('./src/io/css/**', './dist/css'))
```

Comment:
Are you going to keep it this way?

Reply:
For the moment yes, until I can find a better working solution. Will probably minify them before copy though.